main.py

```diff
@@ -2,6 +2,46 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [("George", "Klooney"), ("Kianu", "Reaves"),
+              ("Scarlett", "Keegan"), ("Will", "Smith"),
+              ("Jaden", "Smith"), ("Scarlett", "Johansson")
+              ]
+
+    for genre in genres:
+        Genre.objects.create(
+            name=genre,
+        )
+
+    for actor in actors:
+        Actor.objects.create(
+            first_name=actor[0],
+            last_name=actor[1]
+        )
+
+    Genre.objects.filter(
+        name="Dramma"
+    ).update(name="Drama")
+
+    Actor.objects.filter(
+        last_name="Klooney"
+    ).update(last_name="Clooney")
```

Comment:
What if it will be another Klooney in db? u have to update George according to the task